CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * Added support for configuring cluster-operator's worker thread pool size that is used for various sync and async tasks
 * Add Kafka Quotas plugin with produce, consume, and storage quotas
 * Support pausing reconciliation of KafkaTopic CR with annotation `strimzi.io/pause-reconciliation`
+* The option `enableECDSA` of Kafka CR `authentication` of type `oauth` has been deprecated and is ignored. ECDSA token signature support is now always enabled without the need for Strimzi Cluster Operator installing the BouncyCastle JCE crypto provider. BouncyCastle library is no longer packaged with Strimzi Kafka images.
```

Comment:
Can you move that to the _Changes, deprecations and removals_ section below maybe?